org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/license/ArtifactServerUtil.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright 2019 Association for the promotion of open-source insurance software and for the establishment of open interface standards in the insurance industry (Verein zur Förderung quelloffener Versicherungssoftware und Etablierung offener Schnittstellenstandards in der Versicherungsbranche)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.aposin.licensescout.license;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.List;
+
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Parent;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.aposin.licensescout.archive.Archive;
+import org.aposin.licensescout.util.ILFLog;
+
+/**
+ *
+ */
+public class ArtifactServerUtil {
+
+    private final String mavenCentralBaseUrl;
+    private final int connectTimeoutInMilliseconds;
+    private final boolean cachedCheckAccess;
+
+    /**
+     * Constructor.
+     * 
+     * @param mavenCentralBaseUrl
+     */
+    public ArtifactServerUtil(final String mavenCentralBaseUrl, final int connectTimeoutInMilliseconds,
+            final ILFLog log) {
```

Comment:
The ILFLog here is used to initialize the cachedCheckAccess variable (through the checkAccess method).
Then the addLicensesFromPom method get an ILFLog as parameter too. Is this the same ILFLog object?
If yes, wouldn't be the case to save it as class variable in the constructor and remove it from the two methods parameters?